<ui version="4.0" >
 <class>ChannelEditor</class>
 <widget class="QDialog" name="ChannelEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Channel Editor</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <property name="leftMargin" >
    <number>6</number>
   </property>
   <property name="topMargin" >
    <number>6</number>
   </property>
   <property name="rightMargin" >
    <number>6</number>
   </property>
   <property name="bottomMargin" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Sample Count Limits</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="leftMargin" >
       <number>6</number>
      </property>
      <property name="topMargin" >
       <number>6</number>
      </property>
      <property name="rightMargin" >
       <number>6</number>
      </property>
      <property name="bottomMargin" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="label_8" >
        <property name="text" >
         <string>Min:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sampleMin_" >
          <property name="keyboardTracking" >
           <bool>false</bool>
          </property>
        <property name="toolTip" >
         <string>Minimum sample count for channel</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
        <property name="minimum" >
         <number>-65535</number>
        </property>
        <property name="maximum" >
         <number>65535</number>
        </property>
        <property name="singleStep" >
         <number>10</number>
        </property>
        <property name="value" >
         <number>-8192</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9" >
        <property name="text" >
         <string>Max:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sampleMax_" >
          <property name="keyboardTracking" >
           <bool>false</bool>
          </property>
        <property name="toolTip" >
         <string>Maximum sample count for channel</string>
        </property>
        <property name="frame" >
         <bool>true</bool>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
        <property name="buttonSymbols" >
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
        <property name="minimum" >
         <number>-65535</number>
        </property>
        <property name="maximum" >
         <number>65535</number>
        </property>
        <property name="singleStep" >
         <number>10</number>
        </property>
        <property name="value" >
         <number>8192</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Voltage Limits</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="leftMargin" >
       <number>6</number>
      </property>
      <property name="topMargin" >
       <number>6</number>
      </property>
      <property name="rightMargin" >
       <number>6</number>
      </property>
      <property name="bottomMargin" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>Min:</string>
        </property>
        <property name="textFormat" >
         <enum>Qt::PlainText</enum>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="voltageMin_" >
          <property name="keyboardTracking" >
           <bool>false</bool>
          </property>
        <property name="toolTip" >
         <string>Minimum voltage value for channel</string>
        </property>
        <property name="suffix" >
         <string/>
        </property>
        <property name="minimum" >
         <double>-99.989999999999995</double>
        </property>
        <property name="maximum" >
         <double>99.989999999999995</double>
        </property>
        <property name="value" >
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7" >
        <property name="text" >
         <string>Max:</string>
        </property>
        <property name="textFormat" >
         <enum>Qt::PlainText</enum>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="voltageMax_" >
          <property name="keyboardTracking" >
           <bool>false</bool>
          </property>
        <property name="toolTip" >
         <string>Maximum voltage value for channel</string>
        </property>
        <property name="suffix" >
         <string/>
        </property>
        <property name="minimum" >
         <double>-99.989999999999995</double>
        </property>
        <property name="value" >
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QCheckBox" name="visible_" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string>Check to make channel plots visible</string>
       </property>
       <property name="text" >
        <string>Visible</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="showPeakBars_" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string>Check to show peak bars in channel plots</string>
       </property>
       <property name="text" >
        <string>Show Peak Bars</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox_" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox_</sender>
   <signal>accepted()</signal>
   <receiver>ChannelEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox_</sender>
   <signal>rejected()</signal>
   <receiver>ChannelEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
